### PR TITLE
Rename to f8a_worker

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
TODO: update f8a_worker commit hash variable in the Dockerfile.

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ MAINTAINER Fridolin Pokorny <fridolin@redhat.com>
 
 ENV LANG=en_US.UTF-8 \
     MAVEN_INDEX_CHECKER_PATH='/opt/maven-index-checker' \
-    CUCOSLIB_VERSION=9bbfd8b
+    F8ALIB_VERSION=9bbfd8b
```
Comment (jpopelka) [inline Dockerfile]: I'd suggest `F8WORKER_VERSION` since we are renaming `cucoslib` to `f8a_worker`.

Comment (fridex) [inline Dockerfile]: Will do even in server, thanks!

**Comment by msrb** [comment]
LGTM

**Pull request merged**